extensions/amp-bind/0.1/amp-state.js

```diff
@@ -135,22 +150,39 @@ export class AmpState extends AMP.BaseElement {
       (getSourceOrigin(src) == getSourceOrigin(ampdoc.win.location))) {
       policy = UrlReplacementPolicy.ALL;
     }
-    return batchFetchJsonFor(
-        ampdoc, element, /* opt_expr */ undefined, policy, opt_refresh);
+
+    // If cross-origin="amp-viewer-auth-token-via-post" attribute is present,
+    // the viewer will make a remote xhr POST request with an auth token in the
+    // request body. Requires amp-viewer-assistance extension for auth token.
+    const crossOriginAttr = element.getAttribute('cross-origin');
```

Comment:
Minor: `crossorigin` to align with: https://developer.mozilla.org/en-US/docs/Web/HTML/CORS_settings_attributes

Reply:
Can this be changed in a later PR? Since this is already implemented with some users, we will need some time to deprecate the use of `cross-origin` in favor of `crossorigin`.

Reply:
Done.

As we discussed offline, deprecated `cross-origin` in favor of `crossorigin`. Another PR will come in the future to remove `cross-origin` as well.